scripts/config.pl

```diff
@@ -94,11 +95,34 @@
 _ALT\s*$
 );
 
+# Things that should be disabled in "baremetal"
```

Comment:
These variables should be grouped by configuration, not grouped by direction: `@excluded_baremetal` and `@non_excluded_baremetal` should be next to each other.